### DISCUSSION TITLE
카테캠 제적 위기를 당한 나... 정신을 차리잣!!! (반성문)

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#611] [6. 반성문]
안녕하세욤 ㅎㅎ 경단 프로젝트가 끝나고 최근 좀 더 많이 바빠진 느낌입니다ㅏ!!!

그동안 다른 일들 한다고 카테캠을 좀 대충했더니 엄청난 일이 생겼었습니다.
[나도 모르는 사이에 카테캠 제적 위기에 놓이다](https://stopmin.tistory.com/entry/%EB%82%98%EB%8F%84-%EB%AA%A8%EB%A5%B4%EB%8A%94-%EC%82%AC%EC%9D%B4%EC%97%90-%EC%B9%B4%ED%85%8C%EC%BA%A0-%EC%A0%9C%EC%A0%81-%EC%9C%84%EA%B8%B0%EC%97%90-%EB%86%93%EC%9D%B4%EB%8B%A4)

제적당할 뻔 했습니다요 ㅎㅎ 매니저한테 전화와서 싹싹 빌엇습니다. 이날을 계기로 인턴끝나고 경단하면서 잃을 뻔한 멘탈을 부여잡아야겠다고 다짐했습니다. 인턴 끝나고 나니 뭔가 보상심리(?)가 은근 생겨서 좀 쉬어야지.. 쉬어야지 한 것 같네요. 작년 겨울방학부터 올해 1학기까지 인턴을 하고 카테캠도 병행하면서 그동안 경단 플젝한다고  쉬지 못하고 달린 것 같아서 어쩔 수 없이 생기는 보상심리였는데 개강 한달도 안남은 시점에서 정말 말도안되는 생각이었던 것 같습니다. 그리고 생각해보면 꽤 인턴하면서도 저녁에 많이 쉬었습니다 ㅎㅎ

그리고 각성했습니다! 요즘 갑자기 삘받아서 3학년이지만 이전부터 꼭 원했던 회사에 이번에 원서를 넣으려고 준비중입니다! CS/자격증 몇가지/알고리즘 더더! 필요해서 밤낮으로 준비중입니다!! 조큼 목표가 생기니 해야할 것이 명확해진 것 같아 기분이 좋습니다. 자격증은 두가지 준비중인데 병행하며 PS, CS에 시간을 좀 많이 더 투자해야겠다고 판단했습니다.

아래는 최근 진행하고 있는 자바 스터디인데 정리글 한번 써봤습니다. 써두면 언젠가 읽지 않을까라는 마음으로 정리하면서 공부중입니다!
**자바 스터디**
- [자바 8,9,10,11: 무슨 일이 일어나고 있는가?](https://stopmin.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%EC%9D%B8-%EC%95%A1%EC%85%98-%EC%9E%90%EB%B0%94-891011-%EB%AC%B4%EC%8A%A8-%EC%9D%BC%EC%9D%B4-%EC%9D%BC%EC%96%B4%EB%82%98%EA%B3%A0-%EC%9E%88%EB%8A%94%EA%B0%80)
- [동작 파라미터화 코드 전달하기](https://stopmin.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%EC%9D%B8-%EC%95%A1%EC%85%98-%EB%8F%99%EC%9E%91-%ED%8C%8C%EB%9D%BC%EB%AF%B8%ED%84%B0%ED%99%94-%EC%BD%94%EB%93%9C-%EC%A0%84%EB%8B%AC%ED%95%98%EA%B8%B0)
- [람다 표현식](https://stopmin.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%EC%9D%B8-%EC%95%A1%EC%85%98-%EB%9E%8C%EB%8B%A4-%ED%91%9C%ED%98%84%EC%8B%9D)
- [스트림](https://stopmin.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%EC%9D%B8-%EC%95%A1%EC%85%98-%EC%8A%A4%ED%8A%B8%EB%A6%BCStream)

그리고 알고리즘 ps도 조금 써봤습니다. 그저께부터 하루에 2~3문제씩 푸는 중인데 턱없이 모자란 것 같습니다. 코테 목적으로 프로그래머스를 풀고 있는데 레벨 2 위주로 처리하고 있습니다! 당분간 python으로 진행할 예정입니다! 원래 C++이었는데 안한지 좀 되어서 그런지 신텍스 에러도 엄청 많이 나고, 문제들도 대체적으로 파이썬이 조금 더 유리할 것 같다고 판단했습니다. 또한, 가끔 파이썬 괴수들의 코드가 보이는데 이때까지 파이썬을 조금 무시했는데 그러면 안되겠다 나도 저런 코드 짜보고 싶다!! 라는 생각이 되게 많이 들더라구요. 한달 조금 넘게 남았는데 매일 열심히 하면 잘할 수 있겠죵?? 머 모르겠습니다~~!! 그냥 하는거쥐

- [Lv2. 도넛과 막대 그래프](https://stopmin.tistory.com/entry/%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%98%EB%A8%B8%EC%8A%A4-Lv2-%EB%8F%84%EB%84%9B%EA%B3%BC-%EB%A7%89%EB%8C%80-%EA%B7%B8%EB%9E%98%ED%94%84)
- [Lv2. 요격 시스템](https://stopmin.tistory.com/entry/%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%98%EB%A8%B8%EC%8A%A4-Lv2-%EC%9A%94%EA%B2%A9-%EC%8B%9C%EC%8A%A4%ED%85%9C)
- [Lv1. 가장 많이 받은 선물](https://stopmin.tistory.com/entry/%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%98%EB%A8%B8%EC%8A%A4-Lv1-%EA%B0%80%EC%9E%A5-%EB%A7%8E%EC%9D%B4-%EB%B0%9B%EC%9D%80-%EC%84%A0%EB%AC%BC)

**Comment by shkisme** [comment]
폭주기관차 지민

**Comment by pykido** [comment]
역대급
